src/rdbms/mongoose_rdbms.erl

```diff
@@ -689,7 +702,15 @@ outer_op({sql_transaction, F}, State) ->
 outer_op({sql_dirty, F}, State) ->
     sql_dirty_internal(F, State);
 outer_op({sql_execute, Name, Params}, State) ->
-    sql_execute(outer_op, Name, Params, State).
+    sql_execute(outer_op, Name, Params, State);
+outer_op({sql_execute_wrapped, Name, Params, Wrapper}, State) ->
+    try
+        Wrapper(fun() -> sql_execute(outer_op, Name, Params, State) end)
+    catch
+        _Class:Error ->
+            ?LOG_ERROR(#{what => sql_execute_wrapped_failed, reason => Error}),
```

Comment:
at least log `Name` variable (I think there are examples of LOG_ERROR in the module).